qui/tray/domains.py

```diff
@@ -556,19 +561,31 @@ def have_running_and_all_are_paused(self):
                         return False
         return found_paused
 
-    def add_domain_item(self, _submitter, event, vm, **_kwargs):
+    def add_domain_item(self, _submitter, event, vm_name, **_kwargs):
```

Comment:
You can't do that, events use keyword arguments, so their names needs to match the event signature.

Reply:
One day the day will come when I start remembering this. It's not today.